ccd-definition/FixedLists/GeneratedOrderType.json

```diff
@@ -13,11 +13,18 @@
     "ListElement": "Care order",
     "DisplayOrder": 2
   },
+  {
```

Comment:
Just remembered that I think these are supposed to be ordered alphabetically

Reply:
Nice spot. Fixed